pdf2searchablepdf.sh

```diff
@@ -117,7 +117,7 @@ main() {
         # Ensure file_list_path doesn't yet exist, so that this filename won't get included inside this file itself as
         # part of the list (useful in case this file was left around from a previous run)
         rm -f "$file_list_path" 
-        find "$dir_of_imgs"/* | sort -V > "$file_list_path"
+        find "$dir_of_imgs"/* |grep -v file_list.txt| sort -V > "$file_list_path"
```

Comment:
I don't understand why I would need to explicitly exclude `file_list.txt` when just one line above I'm explicitly deleting it before even calling `find`. How could "file_list.txt" possibly show up in the `find` results when that file was just deleted?